### ISSUE TITLE
Accessibility Issue with Keyboard navigation on 'Help & Support' screen

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4774]
# Intro
This is an accessibility issue I found with the 'Help & Support' screen.

## Issue
Keyboard navigation using the 'tab' key does not work at all for navigating through the options on the screen. This is a violation of WCAG Success Criterion 2.1.1 (Level A) - keyboard. [See](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

## Description
Success Criterion 2.1.1 (Level A) - keyboard requires that all functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

## Impact
People who are blind will be unable to get help and support from the service. People with low vision who find it difficult to track pointers will encounter difficulty, and people with hand tremors who cannot use a computer mouse will also experience navigation difficulties. 

## Fix
The list options would need to be configured to allow for keyboard tab navigation.
